hardware/RF-Test-Guide.md

```diff
@@ -0,0 +1,283 @@
+
+Note that every implementation of an antenna design might require different combinations of inductors and capacitors.
+
+The recommended 3-element PI network for external antenna matching purposes is here:
+
+<div align="center">
+  <img src="files\HW-RF-Test-Guide\matching.png">
+</div>
+
+<div align="center">
+
+<b>Figure 1.  3-element PI type matching network</b> 
+
+</div>
+
+
+## 2.2. Tune and measure
+
+It is recommended to add a pi network for antenna matching as shown in Figure 1. Finding appropriate values for the antenna matching components should be considered an iterative task. Usually it use a Vector Network Analyser(VNA) to tune and measure the antenna matching network as following figure shows. 
+
+<div align="center">
+ <img src="files\HW-RF-Test-Guide\VNA-Test.png">  
+</div>
+
+The following matching steps and tricks are proposed:
+
+- Calibrate your VNA for a frequency range larger than the intended bandwidth of the antenna.
+- Connect an RF coaxial cable to the RF line (for instance by soldering a pigtail to the line). Connect the RF coaxial cable to a VNA to measure the reflection coefficient, S11, looking into the antenna through the matching network.
+
+  a. Be sure to have a good connection to the ground plane to get the best electrical performance and the highest mechanical robustness during the measurement.
+
+  b. Make sure to route the pigtail towards the center of the PCB and then perpendicularly away from the PCB at the center point. This will limit the effect of the cable on the measured data as much as possible.
+
+- Start with no components on the antenna network shown in Figure 1.
+
+  a. The shunt components are not mounted.
+
+  b. The series component is not mounted.
+
+- Using port extension function in the VNA to move the reference point to the footprint of R1 and R2. This is achieved when the locus of the S-parameters in the Smith chart on the VNA have assembled in a point at the right edge of the Smith chart.
+
+- Mount a 0 Ω resistor at R2 in Figure 1.
+
+- Measure reflection coefficient of the center frequency of interest (the frequency half way between the lowest frequency and the highest frequency of the interesting band).
+
+- Using an online matching tool to calculate series and shunt component values to achieve 50 Ω match on the coaxial line. This will give a good starting point and a reasonable result at first attempt.
```

Comment:
Should we provide a few examples of online tools with links?

Reply:
Provided a link here.